src/picongpu/include/simulation_defines/param/speciesAttributes.param

```diff
@@ -58,7 +58,7 @@ value_identifier(bool, radiationFlag, false);
 /** number of electrons bound to the atom / ion
  *
  * value type is float_X to avoid casts during the runtime
- * - float is also reasonable because effective charge numbers are possible
+ * - float_32 is also reasonable because effective charge numbers are possible
  * - only reasonable for ion species if ionization is enabled
```

Comment:
better write:
- `required for ion species if [...]` because someone might find it reasonable even without a certain ionization scheme enabled